client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -93,7 +119,19 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
             onChange={handleChange}
           />
 
-          <Box textAlign="center" marginTop={5}>
+          <Box className={classes.buttonsWrapper} marginTop={5}>
+            <Button
```

Comment:
Optional: As a bonus, it would be nice if we could've componentized the `Demo` login button along with the login helper - for reusability and increased maintainability